### PR TITLE
doc: add documentation for iostat

### PR DESCRIPTION
**Pull request by mogeb**
Signed-off-by: Mohamad Gebai <mgebai@suse.com>

### CODE REVIEW

File: doc/releases/mimic.rst [hunk 1]
```diff
@@ -388,6 +388,11 @@ These changes occurred between the Luminous and Mimic releases.
     via a "pass through" method. See http://docs.ceph.com/docs/master/mgr/restful
     for details.
 
+  - New command to track throughput and IOPS statistics, also available in
+    `ceph -s` and previously in `ceph -w`. To use this command, enable
+    the `iostat` Manager module and invoke it using `ceph iostat`. See the
+    :ref:`iostat documentation <mgr-iostat-overview>` for details.
```
Comment (liewegas) [inline doc/releases/mimic.rst]: Let's add this to mgr_initial_modules so that it is enabled by default.

Comment (liewegas) [reply doc/releases/mimic.rst]: (And let's adjust the language on the module page to reflect that)

Comment (liewegas) [reply doc/releases/mimic.rst]: ...although upgraded clusters won't have it enabled.  Hmm.  Should we enable it automatically during upgrade?  @jcsp

File: doc/releases/mimic.rst [hunk 2]
```diff
@@ -388,6 +388,11 @@ These changes occurred between the Luminous and Mimic releases.
     via a "pass through" method. See http://docs.ceph.com/docs/master/mgr/restful
     for details.
 
+  - New command to track throughput and IOPS statistics, also available in
+    `ceph -s` and previously in `ceph -w`. To use this command, enable
```
Comment (alfredodeza) [inline doc/releases/mimic.rst]: all of these ceph commands need double backticks so that they are properly formatted as monospaced in HTML

**Comment by jcsp** [comment]
Enabling this by default would be reasonable -- I'm fuzzy on the best place to put on-upgrade hooks such as those to enable this module though.  MgrMonitor itself doesn't track feature bits so it's unaware of major upgrades, maybe somewhere in MonMonitor it could notice when a new feature bit gets set across all mons, and trigger the other PaxosService classes to do any "on_major_upgrade" actions they want to.

In the meantime the UX isn't too bad -- anyone who tries to run iostat will get a friendly message telling them how to enable it.

**Comment by mogeb** [comment]
Thanks, I've added this module to `mgr_initial_modules` and to `vstart.sh`. It will still be disabled on upgraded clusters for now though.

**Comment by mogeb** [comment]
@liewegas anything blocking this PR?

**Comment by liewegas** [comment]
looks good!  mind preparing a mimic backport?

**Comment by mogeb** [comment]
@liewegas thanks, will do shortly.

**Pull request merged**